### PR TITLE
871182  Apply themes support code snippet for PdfViewer Control

### PR DESCRIPTION
**Pull request by HemalathaDuraisamy**
871182  Apply themes support code snippet for PdfViewer Control

### CODE REVIEW

File: wpf/Pdf-Viewer/Getting-Started.md [hunk 1]
```diff
@@ -198,5 +198,45 @@ The WPF PdfViewer Control supports various built-in themes. Refer to the below l
...
+public MainWindow()
+{
+    InitializeComponent();
+
+    //Initialize PDF Viewer.
+    PdfViewerControl pdfViewer1 = new PdfViewerControl();
+
+    //Apply the theme to PDFViewer.
+    SfSkinManager.SetTheme(pdfViewer, new Theme() { ThemeName = "FluentDark" });
+    HomeGrid.Children.Add(pdfViewer);
```
Comment (RajaVigneshB) [inline wpf/Pdf-Viewer/Getting-Started.md:215]: Add this line below the PdfViewer initialization

File: wpf/Pdf-Viewer/Getting-Started.md [hunk 2]
```diff
@@ -198,5 +198,45 @@ The WPF PdfViewer Control supports various built-in themes. Refer to the below l
...
 
+Apply the variety of themes support for the PDFViewer control. And use the following example code for how to apply the FluentDark theme to PDFViewer.
+  
+{% tabs %}
+{% highlight c# %}
+
+public MainWindow()
+{
+    InitializeComponent();
+
```
Comment (RajaVigneshB) [inline wpf/Pdf-Viewer/Getting-Started.md:209]: Remove unwanted spaces

File: wpf/Pdf-Viewer/Getting-Started.md [hunk 3]
```diff
@@ -198,5 +198,45 @@ The WPF PdfViewer Control supports various built-in themes. Refer to the below l
...
+    HomeGrid.Children.Add(pdfViewer);
+    pdfViewer.Load(@"../../PDF_Succinctly.pdf");           
+}
+
+{% endhighlight %}
+
+{% highlight vbnet %}
+
+
+    Public Sub New()
```
Comment (RajaVigneshB) [inline wpf/Pdf-Viewer/Getting-Started.md:224]: Remove unwanted spaces.

**Review by RajaVigneshB — CHANGES_REQUESTED** [review]

File: wpf/Pdf-Viewer/Getting-Started.md (hunk 1 above)
Comment (HemalathaDuraisamy) [reply wpf/Pdf-Viewer/Getting-Started.md]: Modified the line of the code

File: wpf/Pdf-Viewer/Getting-Started.md (hunk 2 above)
Comment (HemalathaDuraisamy) [reply wpf/Pdf-Viewer/Getting-Started.md]: Removed the unwanted space

File: wpf/Pdf-Viewer/Getting-Started.md (hunk 3 above)
Comment (HemalathaDuraisamy) [reply wpf/Pdf-Viewer/Getting-Started.md:222]: Removed the unwanted space

**Review by RajaVigneshB — APPROVED** [review]

**Pull request merged**